src/plugins/matrixops/shuffle_operation.cpp

```diff
@@ -68,8 +68,8 @@ namespace phylanx { namespace execution_tree { namespace primitives
     ///////////////////////////////////////////////////////////////////////////
     match_pattern_type const shuffle_operation::match_data =
     {
-        hpx::util::make_tuple("shuffle_operation",
-            std::vector<std::string>{"shuffle_operation(_1)"},
+        hpx::util::make_tuple("shuffle",
```

Comment:
This requires changing the name here as well: https://github.com/STEllAR-GROUP/phylanx/blob/master/phylanx/plugins/matrixops/shuffle_operation.hpp#L69